### DISCUSSION TITLE
Switch to next note after archiving current note

### DISCUSSION THREAD

**Discussion by ningsongshen** [standardnotes/forum#2706] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
When I click 'Archive' on a note, it archives the note but the note remains in the editor. It might be a bit confusing to have it not disappear and be 'archived'. I have to click on the next note so that the current note disappears.

**Describe the solution you'd like**
The editor should switch to the next note in the list (kind of like 'removing' the note).

**Additional context**
Other note apps I've used previously have the note disappear when 'archive' is clicked (and immediately switch to the next note). Similar to email clients too.

**Comment by JaspalSuri** [comment]
I personally prefer how it currently is, but I'll let @zsoltszilvai chime in. :)

**Comment by fineseasalt** [comment]
Added it to my list, will do some research and see which solution drives the most value to our users. Thanks a lot for the feedback! 👍
